Application/EdFi.Ods.Api.IntegrationTestHarness/Standard/4.0.0/MsSql/Structure/Ods/0010-Create-StudentWithCTE-Auth-View.sql

```diff
@@ -0,0 +1,14 @@
+CREATE OR ALTER VIEW auth.StudentWithCTECourseEnrollments
```

Comment:
These views for the custom-view based authorization required versions specific to the v4.x and v5.x Standards due to slight model changes.